package.json

```diff
@@ -22,14 +22,17 @@
     "url": "git://github.com/matomo-org/matomo-tracker.git"
   },
   "scripts": {
-    "test": "mocha --reporter spec"
+    "test": "npm run mocha && npm run eslint",
+    "mocha": "mocha --reporter spec",
+    "eslint": "eslint index.js test/index.test.js"
```

Comment:
you can use `eslint --fix` to automatically fix formatting issues (e.g. whitespace). Very handy.

Reply:
Nice, I didn't know that.
But in this case (travis) it isn't needed.